modules/ROOT/pages/tutorials.adoc

```diff
@@ -0,0 +1,11 @@
+= Tutorials
+
+These are some tutorials focused on helping you get started with basic provisioning scenarios for Fedora CoreOS.
```

Comment:
I would suggest:
The following tutorials are focused on helping you getting started with provisioning scenarios for Fedora CoreOS.

Reply:
https://github.com/coreos/fedora-coreos-docs/blob/b8c49f9570196ebbed0ac8c7c866b9e7f17d8879/modules/ROOT/pages/tutorials.adoc fixed.